### ISSUE TITLE
Several UI components are not keyboard accessible

### ISSUE THREAD

**Issue by daltojohnso** [metabase/metabase#15551]
Various UI input components we maintain are not keyboard accessible, meaning forms cannot be filled out without trackpad/mouse usage. Found some obvious problems with the following components:

- [x] `Button`
- [x] `Radio`
- [ ] `Select`
- [x] `Toggle`
- [ ] `AccordionList`
- [ ] `Calendar`
- [ ] `ColorPicker`
- [ ] `TokenField`

Will post videos of each in separate comments.

**Comment by daltojohnso** [comment]
1. `Button` component has no outline to signify an "active" state when you tab to it
http://localhost:3000/_internal/components/button

https://user-images.githubusercontent.com/13057258/114235568-39626c00-9935-11eb-80aa-70482cc0ad75.mov

**Comment by daltojohnso** [comment]
2. `Radio` can't be tabbed to
http://localhost:3000/_internal/components/radio

https://user-images.githubusercontent.com/13057258/114235576-3bc4c600-9935-11eb-80d5-81ae49bb4b45.mov

**Comment by daltojohnso** [comment]
3. `Select` can't be tabbed to
http://localhost:3000/_internal/components/select

https://user-images.githubusercontent.com/13057258/114235581-3e272000-9935-11eb-926a-20a7284895fb.mov

**Comment by daltojohnso** [comment]
4. `Toggle` can’t be tabbed to
http://localhost:3000/_internal/components/toggle

https://user-images.githubusercontent.com/13057258/114235587-40897a00-9935-11eb-9fd5-95655a2566a9.mov

**Comment by daltojohnso** [comment]
5. `AccordionList` can’t be tabbed to
http://localhost:3000/_internal/components/accordionlist

https://user-images.githubusercontent.com/13057258/114235598-42ebd400-9935-11eb-94fd-1092e6258e53.mov

**Comment by daltojohnso** [comment]
6. `Calendar` can’t be tabbed to
http://localhost:3000/_internal/components/calendar

https://user-images.githubusercontent.com/13057258/114235602-44b59780-9935-11eb-8d62-1e143ea30308.mov

**Comment by daltojohnso** [comment]
7. `ColorPicker` can’t be tabbed to
http://localhost:3000/_internal/components/colorpicker

https://user-images.githubusercontent.com/13057258/114235607-4717f180-9935-11eb-934b-d9b1e7d6012c.mov

**Comment by daltojohnso** [comment]
8. `TokenField` can be tabbed to and is partially keyboard accessible, but tabbing away causes a value to be selected automatically
http://localhost:3000/_internal/components/tokenfield

https://user-images.githubusercontent.com/13057258/114235615-497a4b80-9935-11eb-801e-88f3b2f5e0cc.mov

**Comment by flamber** [comment]
Related to #1462 and #5686 and #12807

**Comment by ranquild** [comment]
Do we have a mockup from designers for all this states?
